### PR TITLE
HCAP-1076: change RoS site

### PR DESCRIPTION
**Pull request by alva-fresh**
https://freshworks.atlassian.net/browse/HCAP-1076

Adding a new form to change RoS site

### CODE REVIEW

File: client/src/components/modal-forms/NewPSIForm.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { Button } from '../generic';
+import { Box } from '@material-ui/core';
+import { RenderTextField, RenderSelectField } from '../fields';
+import { Field, Formik, Form as FormikForm } from 'formik';
+import { CreatePSISchema, healthAuthorities } from '../../constants';
+
+export const NewPSIForm = ({ initialValues, onSubmit, onClose }) => {
+  return (
+    <Formik initialValues={initialValues} validationSchema={CreatePSISchema} onSubmit={onSubmit}>
+      {({ submitForm, values }) => (
```
Comment (taraepp) [inline client/src/components/modal-forms/NewPSIForm.js:12]: See SonarCloud scan (code smell: remove unused values param or rename to _values)- I think that could be cleaned up!

**Review by taraepp — CHANGES_REQUESTED** [review]
Just the sonarcloud code smell, looks good to me!

File: client/src/components/modal-forms/NewPSIForm.js (hunk 1 above)
Comment (arranfw) [reply client/src/components/modal-forms/NewPSIForm.js:12]: I think `_values` is a mistake for interpreting this bit as function parameters, in this case where it's a destructured object, removal is the best option if it's unused

**Review by taraepp — DISMISSED** [review]
Looks good to me!

**Review by taraepp — APPROVED** [review]
👍

**Pull request merged**